README-docker.md

```
@@ -0,0 +1,151 @@
+
+<h1 align="center">m4b-merge</h1>
+
+<div align="center">
+
+[![Status](https://img.shields.io/badge/status-active-success.svg)]()
```

markdownlint-cli2 (0.20.0)

[warning] No empty links

(MD042, no-empty-links)